app/src/renderer/components/common/ShortAddress.vue

```diff
@@ -0,0 +1,79 @@
+<template lang="pug">
+.short-address
+  .address(@click.prevent.stop="copy" v-tooltip.top="address")#address {{ shortAddress }}
+  .success(:class="{active:showSuccess}")
+    i.material-icons check
+    span Copied
+</template>
+
+<script>
+import { clipboard } from "electron"
+export default {
+  name: `short-address`,
+  props: [`address`],
+  data: () => ({
+    showSuccess: false
+  }),
+  computed: {
+    shortAddress({ address } = this, length = 4) {
```

Comment:
question: do we always want to show the last 4 chars ? maybe getting the `length` value from the props is a better option ?

Reply:
right now, we are going to show only the last 4 - but it will be easy to change it if we decide we should.